api/app/Providers/AppServiceProvider.php

```diff
@@ -32,6 +34,27 @@ public function boot()
             'personalExperience' => \App\Models\PersonalExperience::class,
             'workExperience' => \App\Models\WorkExperience::class,
         ]);
+
+        // DB::whenQueryingForLongerThan(50, function (Connection $connection, QueryExecuted $event) {
+        //     // Notify development team...
+        //     Log::debug(
+        //         [
+        //             'sql' => $event->sql,
+        //             'bindings' => $event->bindings,
+        //             'milliseconds' => $event->time,
+        //         ]
+        //     );
+        // });
+
+        // DB::listen(function ($query) {
+        //     if ($query->time > 20) {
+        //         Log::warning('Query exceeded 20 milliseconds -', [
+        //             'sql' => $query->sql,
+        //             'milliseconds' => $query->time,
+        //         ]);
+        //     }
+        // });
```

Comment:
An idea I found elsewhere to use the familiar listen method. This logs the specific SQL statements that take longer than a certain time threshold but not sure if it offers much insight as it might be confusing to comb through. As you don't know the operation context so you can't do much with the information. 

![Screenshot 2024-01-17 172016](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/1271e183-fe9c-4532-8b17-179538731318)

Reply:
Good to know!  This could be useful for troubleshooting.  Could you leave this in the code, commented out?

Reply:
Yeah sure, will do
https://github.com/GCTC-NTGC/gc-digital-talent/pull/9045/commits/28cb381993b69da19a5b5b5cceb8d323add30f5f